dd-sdk-android/src/test/kotlin/com/datadog/android/tracing/TracingInterceptorNonDdTracerTest.kt

```diff
@@ -168,6 +168,11 @@ internal open class TracingInterceptorNonDdTracerTest {
 
         val mediaType = forge.anElementFrom("application", "image", "text", "model") +
             "/" + forge.anAlphabeticalString()
+        fakeLocalHosts = forge.aMap {
+            forge.aStringMatching(TracingInterceptorTest.HOSTNAME_PATTERN) to setOf(
+                TracingHeaderType.DATADOG
```

Comment:
probably we should add some simple tests which check that if host is associated with multiple tracing header types, they all co-exist well in the final request (unless I missed them).

Upd: encountered some such later.